Flaky unit test

This test failed on travis:
```
--- FAIL: TestAnnouncerAnnounceUpdatesInterval (0.76s)
    require.go:312: 
        	Error Trace:	announcer_test.go:45
        	            				announcer_test.go:111
        	Error:      	Tick timed out
        	Test:       	TestAnnouncerAnnounceUpdatesInterval
FAIL
coverage: 82.6% of statements
FAIL	github.com/uber/kraken/lib/torrent/scheduler/announcer	0.779s
```

--- a/lib/torrent/scheduler/announcer/announcer_test.go
+++ b/lib/torrent/scheduler/announcer/announcer_test.go
@@ -15,6 +15,7 @@ package announcer
 
 import (
 	"errors"
+	"runtime"
 	"testing"
 	"time"
 
@@ -39,6 +40,7 @@ func newMockEvents() *mockEvents {
 func (e *mockEvents) AnnounceTick() { e.tick <- struct{}{} }
 
 func (e *mockEvents) expectTick(t *testing.T) {
+	runtime.Gosched()
 	select {
 	case <-e.tick:
 	case <-time.After(500 * time.Millisecond):
@@ -47,6 +49,7 @@ func (e *mockEvents) expectTick(t *testing.T) {
 }
 
 func (e *mockEvents) expectNoTick(t *testing.T) {
+	runtime.Gosched()
 	select {
 	case <-e.tick:
 		require.FailNow(t, "Unexpected tick")

